pom.xml

```diff
@@ -82,18 +83,40 @@
       <resource>
         <directory>src/main/resources</directory>
       </resource>
+      <!-- packaged into zip -->
+      <resource>
+        <directory>src/main/plugin/iso19139.ca.HNAP</directory>
+      </resource>
     </resources>
-
     <testResources>
       <testResource>
         <directory>src/test/resources/org/fao/geonet</directory>
       </testResource>
-      <testResource>
-        <directory>src/main/plugin/iso19139.ca.HNAP</directory>
-      </testResource>
     </testResources>
 
     <plugins>
+      <plugin>
+        <artifactId>maven-jar-plugin</artifactId>
+        <configuration>
+          <excludes>
+            <!-- ignore schema-plugin content -->
+            <exclude>convert/**</exclude>
+            <exclude>formatter/**</exclude>
+            <exclude>index-fields/**</exclude>
+            <exclude>layout/**</exclude>
+            <exclude>loc/**</exclude>
+            <exclude>present/**</exclude>
+            <exclude>process/**</exclude>
+            <exclude>sample-data/**</exclude>
+            <exclude>schema/**</exclude>
+            <exclude>schematron/**</exclude>
+            <exclude>templates/**</exclude>
+            <exclude>*.xsl</exclude>
+            <exclude>*.xml</exclude>
+            <exclude>*.xsd</exclude>
+          </excludes>
+        </configuration>
+      </plugin>
```

Comment:
I had just come to the same conclusion myself, now using includes to only grab the normal jar content.

And yes we expect this to look like a normal resource folder for IntelliJ (since it is a normal resource folder - it is just that we package this thing into a `jar` and `zip` for distribution.

Reply:
@jodygarnett 
I looked into the include as well and that should work but it still requires a change when a new folder is added. I find it easier to simply remove the <plugin> code altogether and use the suggested code that I included which includes the resource so that it will be in the zip(and show up properly in intellij). But then excludes all files so that it will not be in the jar.  I think this would be cleaner and will not cause issues when new data is added.

Reply:
@ianwallen I did not quite understand your code, since the assembly for the zip already has that logic:
``` 
<fileSets>
    <fileSet>
      <directory>src/main/plugin/</directory>
      <outputDirectory></outputDirectory>
      <useDefaultExcludes>true</useDefaultExcludes>
    </fileSet>
  </fileSets>
```

So what I am missing from your approach is how to ensure the tests can see the plugin contents?

Note if the plugin folder is not used by the test cases we can solve this very quickly, by removing `src/main/plugin/iso19139.ca.HNAP` folder as resource and as a testResource. 

It will still be packaged in the assembly above.

Reply:
@jodygarnett 
I knew there was something basic that I was missing.  Locally I don't run the tests. 

Changes look good.